tools/build-debian-copyright

```diff
@@ -0,0 +1,168 @@
+        return 'Apache-2.0'
+    if mod == 'redux':
+        # our tarball only ships the minified version, original source is at
+        # https://github.com/reactjs/redux; as it's "MIT" we don't legally
+        # require to ship the preferred form of modification
+        return 'MIT'
+
+    raise SystemError('Could not determine license from %s' % moddir)
+
+
+def module_copyright(moddir):
+    '''Return Copyrights for given module'''
+
+    mod = os.path.basename(moddir)
+    copyrights = set()
+    try:
+        out = subprocess.check_output(['env', '-u', 'LANGUAGE', 'LC_ALL=C.UTF-8', 'grep', '-hri', 'copyright.*[1-9][0-9]\+'],
+                                      cwd=moddir, universal_newlines=True)
+        for l in out.splitlines():
+            # ignore compiled .css.map files
+            if l.startswith('{"version":') or '*//*!' in l:
+                continue
+            # weed out some noise
+            if 'grunt.template.today' in l or 'copyright-mark' in l or '@font-face' in l or 'Binary file' in l:
+                continue
+            l = l.replace('<copyright>', '').replace('</copyright>', '')
+            l = l.replace('&copy;', '(c)').replace('copyright = ', '')
+            l = re.sub('@license.*Copyright', '', l)
+            l = l.strip(' \t*/\'|,')
+            if not l.endswith('Inc.'):
+                # avoids some duplicated lines which only differ in trailing dot
+                l = l.strip('.')
+            if l.startswith("u'") or l.startswith('u"'):  # Python unicode prefix
+                l = l[2:]
+            if 'Fixes #' in l:
+                continue  # this is from a changelog
+            # some noise fine-tuning for specific packages
+            if mod == 'bootstrap' and ('https://github.com' in l or l == 'Copyright 2015'):
+                continue
+            if mod == 'patternfly':
+                l = re.sub(' and licensed.*', '', l)
+
+            copyrights.add((' ' + l).decode('UTF-8'))  # space prefix for debian/copyright RFC822 format
+    except subprocess.CalledProcessError:
+        pass
+
+    # missing copyrights
+    if mod == 'angular-bootstrap':
+        # only committer is Wesley Cho, so assume his copyright
+        # https://github.com/angular-ui/bootstrap-bower/issues/74
+        copyrights.add('(C) 2015-2017 Wesley Cho')
+    if mod == 'kubernetes-object-describer':
+        # only committer is Jessica Forrester, working for Red Hat
+        # https://github.com/kubernetes-ui/object-describer/issues/35
+        copyrights.add('(C) 2015-2017 Red Hat, Inc.')
+    if mod == 'redux':
+        # our tarball only ships the minified version, original source is at
+        # https://github.com/reactjs/redux
+        copyrights.add('(c) 2015-present Dan Abramov')
+
+    if not copyrights:
+        raise SystemError('Did not find any copyrights in %s' % moddir)
+    return copyrights
+
+#
+# main
+#
+
+if len(sys.argv) != 2:
```

Comment:
Completely agreed, this was more or less a leftover. I also forgot to add it to `tools/static-code-tests`. Both done now.